src/style.css

```diff
@@ -1 +1,86 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.search-bar--big,
+.search-bar--small {
+  margin-top: 20px;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  gap: 20px;
+}
+
+.search-bar {
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  box-shadow: 0 4px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search-bar__input--big {
+  background-size: 19px 19px;
+  background-position: 26px center;
+  padding-left: 62px;
+  font-size: 16px;
+  height: 70px;
+}
+
+.search-bar__input--small {
+  background-position: 13px center;
+  background-size: 11px 11px;
+  font-size: 14px;
+  padding-left: 33px;
+  height: 42px;
+}
+
+.search-bar__input::placeholder {
+  color: #3d4e61;
+}
+
+.search-bar__input:hover {
+  box-shadow: 0 1px 8px 0 rgba(0, 0, 0, 0.25);
+}
+
+.search-bar__input:focus {
+  font-family: Avenir-Heavy, sans-serif;
```

Comment:
Here you're changing the font family on focus, which can lead to a jarring experience for the user. It's better to keep the font family consistent across different states.